src/reformat.c

```diff
@@ -313,6 +333,21 @@ avifResult avifImageRGBToYUV(avifImage * image, const avifRGBImage * rgb)
                             yuvBlock[bI][bJ].y = 0.5f * rgbPixel[1] + 0.25f * (rgbPixel[0] + rgbPixel[2]);
                             yuvBlock[bI][bJ].u = 0.5f * rgbPixel[1] - 0.25f * (rgbPixel[0] + rgbPixel[2]);
                             yuvBlock[bI][bJ].v = 0.5f * (rgbPixel[0] - rgbPixel[2]);
+#if defined(AVIF_ENABLE_EXPERIMENTAL_YCGCO_R)
+                        } else if (state.mode == AVIF_REFORMAT_MODE_YCGCO_RE || state.mode == AVIF_REFORMAT_MODE_YCGCO_RO) {
+                            // Formulas from JVET-U0093.
+                            const int bitOffset = (state.mode == AVIF_REFORMAT_MODE_YCGCO_RE) ? 2 : 1;
+                            const int maxValue = (1 << (state.yuvDepth - bitOffset)) - 1;
+                            const int R = avifRoundf(AVIF_CLAMP(rgbPixel[0] * rgbMaxChannelF, 0, maxValue));
+                            const int G = avifRoundf(AVIF_CLAMP(rgbPixel[1] * rgbMaxChannelF, 0, maxValue));
+                            const int B = avifRoundf(AVIF_CLAMP(rgbPixel[2] * rgbMaxChannelF, 0, maxValue));
```

Comment:
Nit: In lines 341-343, change `0, maxValue` to `0.0f, rgbMaxChannelF` so that the constants have the correct type (float).

Reply:
Done